app/models/equality_and_diversity_feature_metrics.rb

```diff
@@ -0,0 +1,54 @@
+class EqualityAndDiversityFeatureMetrics
+  include ActionView::Helpers::NumberHelper
+
+  def response_rate(
+    start_time,
+    end_time = Time.zone.now.end_of_day
+  )
+    success_count = application_forms_with_feedback(start_time, end_time).count.to_f
+    fail_count = application_forms_with_no_feedback(start_time, end_time).count.to_f
+    return nil if (fail_count + success_count).zero?
+
+    success_count / (fail_count + success_count)
+  end
+
+  def formatted_response_rate(
+    start_time,
+    end_time = Time.zone.now.end_of_day
+  )
+    format_as_percentage(response_rate(start_time, end_time))
+  end
+
+private
+
+  def format_as_percentage(ratio)
+    return 'n/a' if ratio.nil?
+
+    percentage = number_with_precision(
+      100.0 * ratio,
+      precision: 1,
+      strip_insignificant_zeros: true,
+    )
+    "#{percentage}%"
+  end
+
+  def application_forms_with_feedback(start_time, end_time)
+    ApplicationForm
+      .where(
+        recruitment_cycle_year: RecruitmentCycle.current_year,
```

Comment:
Do we always scope the results on the feature metric dashboard to the current cycle? I'm just wondering if we should or not. I wouldn't have thought that we would as it means that we wouldn't really get anything out of it for the first 30 days of a new cycle.

Wdyt?

Reply:
@davidgisbey yes rightly or wrongly this is the way all the metrics are scoped right now. Something worth looking at across the board before we start the next cycle.

Reply:
Perhaps it's worth a discussion with  Support Users as to what would be most useful?

I guess current cycle makes sense from a perspective of keeping the info light and relevant. If we scoped it to across all cycles it would quickly beome not as useful.

Maybe there's scope for including `last_cycle` metrics as a comparison point? Appreciate the dashboard is pretty chunky atm and is continuing to grow.